src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -33,9 +33,11 @@ const getFilteredChangeRequests = (changeRequests: ChangeRequest[], user: Authen
 interface ChangeRequestDropdownProps {
   control: Control<any, any>;
   name: string;
+  errors: FieldErrors<ChangeRequest>;
+  errorMessage?: FieldError;
```

Comment:
We wanna minimize the number of inputs/arguments in the component as much as we can. We can delete `errorMessage` field, and replace `errorMessage` with `errors.crId` on line 69 in this file.